### PR TITLE
update experimental features and basic features for 6.3

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update basic features and experimental features for v6.3.0. The changes are listed as follows:

- 新增基本功能：
  - Range COLUMNS 分区
  - Range INTERVAL 分区

-  experimental 转 GA：
   - JSON 类型
   - JSON 函数
   - EXCHANGE PARTITION 功能
   - 提升多并发下的数据扫描性能

- 新增 experimental：
  - Range INTERVAL 分区
  - 元数据锁
  - 添加索引加速

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: experimental-features.md [hunk 1]
```diff
@@ -35,7 +34,10 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md)。（v6.1.0 实验特性）
++ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [Range INTERVAL 分区](/partitioned-table.md#range-interval-partitioning) （v6.3.0 实验特性）
++ [PARTITION BY RANGE COLUMNS (column_list)](/partitioned-table.md#range-columns-分区) （v6.3.0 实验特性）
```
Comment (hfxsd) [inline experimental-features.md:39]:
```suggestion
+ [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) （v6.3.0 实验特性）
```

File: basic-features.md [hunk 2]
```diff
@@ -10,127 +10,127 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
+| [CM-Sketch](/statistics.md)                               | 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 |  默认关闭  |  默认关闭  |  Y  |  Y  |  Y  | Y        |
+| [直方图](/statistics.md)                                  |   Y  |   Y   |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| 扩展统计信息（多列）                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
+| [统计反馈](/statistics.md#自动更新)                       | 已废弃 | 已废弃 | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [统计信息自动更新](/statistics.md#自动更新) |   Y |   Y  |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [动态剪裁](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N |
```
Comment (hfxsd) [inline basic-features.md:133]: 和 L121 重复。要不要把这里的删掉？

File: basic-features.md [hunk 3]
```diff
@@ -10,127 +10,127 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| 分区                                             | 6.3 | 6.2 | 6.1 | 6.0| 5.4          |   5.3    |   5.2    |   5.1    |   5.0    | 4.0 |
+| ------------------------------------------------------------ | :--: | :--: | :--: | :--: | ------------ | :----------: | :----------: | :----------: | :----------: | :-----: |
+| [Range 分区](/partitioned-table.md)                  | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |    Y    |
+| [Hash 分区](/partitioned-table.md)                   | Y  | Y  | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |    Y    |
+| [List 分区](/partitioned-table.md)                   | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |    N    |
+| [List COLUMNS 分区](/partitioned-table.md)           | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |    N    |
+| [`EXCHANGE PARTITION`](/partitioned-table.md)               | Y  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |    N    |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |      N       |    N    |
```
Comment (hfxsd) [inline basic-features.md:122]:
```suggestion
| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | N | N | N | N | N |N | N | N |      N       |    N    |
| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | 实验特性 | N | N | N | N | N |N | N | N |      N       |    N    |
```

File: basic-features.md [hunk 4]
```diff
@@ -10,127 +10,127 @@ aliases: ['/docs-cn/dev/basic-features/']
...
```
Comment (shichun-0415) [inline basic-features.md:121]:
```suggestion
| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |      N       |    N    |
```

File: basic-features.md (hunk 2 above)
Comment (shichun-0415) [inline basic-features.md:133]:
```suggestion
```

Comment (shichun-0415) [reply basic-features.md]: Fixed in [7bb0f09](https://github.com/pingcap/docs-cn/pull/11423/commits/7bb0f09e4ade7b1c4aec9664bf151e80e9d580bf)

File: experimental-features.md [hunk 5]
```diff
@@ -35,7 +34,10 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md)。（v6.1.0 实验特性）
++ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [Range INTERVAL 分区](/partitioned-table.md#range-interval-partitioning) （v6.3.0 实验特性）
++ [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) （v6.3.0 实验特性）
```
Comment (hfxsd) [inline experimental-features.md:39]:
```suggestion
```

File: experimental-features.md [hunk 6]
```diff
@@ -35,7 +34,9 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md)。（v6.1.0 实验特性）
++ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [Range INTERVAL 分区](/partitioned-table.md#range-interval-partitioning) （v6.3.0 实验特性）
```
Comment (shichun-0415) [inline experimental-features.md:38]:
```suggestion
+ [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) （v6.3.0 实验特性）
```

File: experimental-features.md [hunk 7]
```diff
@@ -35,7 +34,9 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md)。（v6.1.0 实验特性）
++ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) （v6.3.0 实验特性）
```
Comment (Oreoxmt) [inline experimental-features.md:38]:
```suggestion
+ [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区)（v6.3.0 实验特性）
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**